pkgs/by-name/la/lager/package.nix

```diff
@@ -0,0 +1,34 @@
+{ callPackage, stdenv, lib
+, fetchFromGitHub
+, boost, immer, zug
+, pkg-config, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lager";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "arximboldi";
+    repo = "lager";
+    rev = "v${version}";
+    hash = "sha256-KTHrVV/186l4klwlcfDwFsKVoOVqWCUPzHnIbWuatbg=";
+  };
+  buildInputs = [
+    boost
+    immer
+    zug
+  ];
+  nativeBuildInputs = [
+    cmake
+  ];
+  cmakeFlags = [
+    "-Dlager_BUILD_TESTS=OFF"
```

Comment:
Is there a specific reason to not build and run the tests? Usually we try to whenever possible, if it's not possible it should be documented with a comment somewhere in this derivation.

Reply:
Tests enabled.